[2023-01-06 12:40:03,479] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Demo.Dag_Test scheduled__2023-01-06T12:39:00+00:00 [queued]>
[2023-01-06 12:40:03,501] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Demo.Dag_Test scheduled__2023-01-06T12:39:00+00:00 [queued]>
[2023-01-06 12:40:03,502] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-06 12:40:03,503] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-06 12:40:03,503] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-06 12:40:05,168] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): Dag_Test> on 2023-01-06 12:39:00+00:00
[2023-01-06 12:40:05,174] {standard_task_runner.py:52} INFO - Started process 8174 to run task
[2023-01-06 12:40:05,178] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Demo', 'Dag_Test', 'scheduled__2023-01-06T12:39:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/demonstration.py', '--cfg-path', '/tmp/tmpg9s6monn', '--error-file', '/tmp/tmpz8ue63tj']
[2023-01-06 12:40:05,179] {standard_task_runner.py:80} INFO - Job 9: Subtask Dag_Test
[2023-01-06 12:40:05,979] {task_command.py:369} INFO - Running <TaskInstance: Demo.Dag_Test scheduled__2023-01-06T12:39:00+00:00 [running]> on host 6ef47e34797a
[2023-01-06 12:40:09,117] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=Dag_Test
AIRFLOW_CTX_EXECUTION_DATE=2023-01-06T12:39:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-06T12:39:00+00:00
[2023-01-06 12:40:09,118] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-06 12:40:09,119] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Hello"']
[2023-01-06 12:40:09,640] {subprocess.py:85} INFO - Output:
[2023-01-06 12:40:09,646] {subprocess.py:92} INFO - Hello
[2023-01-06 12:40:09,647] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-06 12:40:09,676] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Demo, task_id=Dag_Test, execution_date=20230106T123900, start_date=20230106T124003, end_date=20230106T124009
[2023-01-06 12:40:09,730] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-06 12:40:09,760] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
